src/critic.lisp

```diff
@@ -140,9 +140,23 @@
     (values problems-count)))
 
 
+(defparameter pattern-names
+  (mapcar #'(lambda (rule) (string-downcase (symbol-name rule)))
+          (lisp-critic:get-pattern-names))
```

Comment:
Did you know STRING-DOWNCASE can work not only with strings, but also with characters and symbols?
This code can be just `(mapcar #'string-downcase (lisp-critic:get-pattern-names))`.

Reply:
> Did you know STRING-DOWNCASE can work not only with strings, but also with characters and symbols?

Thanks for the advice. I actually didn't knew this ;)